eox_core/api/v1/serializers.py

```diff
@@ -7,6 +7,7 @@
 from collections import OrderedDict
 
 from django.conf import settings
+from django_countries.serializer_fields import CountryField  # pylint: disable=import-error
```

Comment:
why we need to disable the import error? didn't you add django-countries?

Reply:
You are right, thanks